bin/upgrading/update-db-general.pl

```diff
@@ -478,11 +478,11 @@
 )
 EOC
 
-post_create("supportpointsum",
-            "sqltry" => "INSERT IGNORE INTO supportpointsum (userid, totpoints, lastupdate) " .
-            "SELECT userid, SUM(points), 0 FROM supportpoints GROUP BY userid");
+post_create( "supportpointsum",
+          "sqltry" => "INSERT IGNORE INTO supportpointsum (userid, totpoints, lastupdate) "
+        . "SELECT userid, SUM(points), 0 FROM supportpoints GROUP BY userid" );
```

Comment:
Okay, I'm not sure if this is misleading or I just don't like it, but this sort of makes it look like the "SELECT" string is going to be appended to "sqltry" at first glance.